Jenkinsfile

```diff
@@ -1,17 +1,46 @@
 projectBaseName = "infra-elasticsearch"
-elasticsearch_version = "7.5.1-erento-002"
 
-imageName = "${projectBaseName}:${elasticsearch_version}"
+shortLabel = projectBaseName.size() >= 10 ? projectBaseName.substring(0, 10) : projectBaseName
+buildLabel = "${shortLabel}-${UUID.randomUUID().toString()}"
 
-node {
-    stage("checkout") {
-        checkout scm
-    }
+imageVersion = "002"
```

Comment:
Yes, we have the same approach in https://github.com/erento/infra-elasticsearch. The image should (must) not change often and we would like to keep this manual step to avoid building new images for no reason.